chart/README.md

```diff
@@ -101,6 +101,8 @@ You can set up the credentials, nodeSelector, volume sizes (default volumes crea
 | `promscale.image`                        | Docker image to use for the Connector                 | `timescale/promscale:0.1.0-alpha.2` |
 | `promscale.connection.dbName`            | Database to store the metrics in                      | `postgres`  |
 | `promscale.connection.user`              | User used for connection to db | `postgres` |
+| `promscale.connection.dbURI.name`         | DB uri name is used as a key for secret which will hold the db URI value | `db-uri` |
+| `promscale.connection.dbURI.secretTemplate` | The template for generating the name of a secret object which will hold the db URI | `` |
```

Comment:
I don't see where these two variables are actually used.

Reply:
As tobs is a CLI we are assigning these values in `helmInstall.go` I mentioned the fields here because if someone wants to use them while using helm chart. This will be helpful.